### PR TITLE
Add French translation 🇫🇷

### PR DESCRIPTION
**Pull request by excessivecoding**
- [x] getting-started.md
- [x] reactivity.md
- [x] rendering.md
- [x] server.md
- [x] api.md

Not in this PR:
- [ ] comparaison.md (deleguated to @MidouWebDev)
- [ ] faq.md (deleguated to @MidouWebDev)

### CODE REVIEW

**Comment by MidouWebDev** [comment]
@davedbase @amoutonbrady Can you review this PR and merge it so that i can start translation the 3 last files ?

**Comment by MidouWebDev** [comment]
@davedbase @amoutonbrady Can you please merge this PR, so that i can start working on the other files ?
I am ready to contribute !

**Comment by amoutonbrady** [comment]
@davedbase I've quickly red through the code and it's actually quite well written! It's always a bit awkward around the technical terms but I guess that's normal. I, myself I'm rather poor at translating English to French and vice versa due to the way I learnt English...

Isn't there a way for people to contribute to an already opened up PR? If no, then I think this can be merged as is so that people like @MidouWebDev can go ahead and participate in the remaining files.

I'll do a final, more thorough pass when all the files are translated :)

**Comment by MidouWebDev** [comment]
@amoutonbrady please merge this PR, it will be good for the others so that they can appear in the contributors leaderboard because if they contribute to the forked repo, they may not appear and their commits go to the PR creator so to the Fork owner.

So please merge the PR, thanks a lot !

**Comment by excessivecoding** [comment]
Thanks for the comment @amoutonbrady , I took a lot of technical terms straight from Vue and React documentation which helps a lot.

I'll finish translating `api.md` today, and then we will be good to merge if you guys would rather do it like that, but as I said in Discord @MidouWebDev , if you contribute to the fork, your contribution will appear if the commit are not squash (which can be avoided).

**Comment by davedbase** [comment]
@xstevenyung sounds great. MidouWebDev contacted me on Discord and I suggested he open a PR of his own to focus on comparison.md and faq.md. So as you wrap up api.md and he does the other two we should land the whole French translation nicely.

**Comment by MidouWebDev** [comment]
Yeah so once @xstevenyung 's PR is merged i am going to fork the solid-docs repo and contribute to it by translating the last 2 files and then i will create a PR.

File: fr/guides/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+description: Un guide sur comment débuter sur Solid.
+sort: 0
+---
+
+# Pour commencer
+
+## Essayer Solid
+
+La manière la plus simple de commencer avec Solid est d'essayer en ligne. Notre REPL sur https://playground.solidjs.com est la meilleure façon d'expérimenter. Mais aussi sur https://codesandbox.io/ où vous pouvez modifier nos différents exemples.
+
```
Comment (lukechu10) [inline fr/guides/getting-started.md]:
```suggestion
La manière la plus simple de commencer avec Solid est d'essayer en ligne. Notre REPL sur https://playground.solidjs.com est la meilleure façon d'expérimenter, ainsi que sur https://codesandbox.io/ où vous pouvez modifier nos différents exemples.
```

File: fr/guides/getting-started.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+const [last, setLast] = createSignal("Bourne");
+
+createEffect(() => console.log(`${first()} ${last()}`));
+```
+
+Vous pouvez en apprendre plus sur [le système de réactivité de Solid](https://www.solidjs.com/docs/latest#reactivity) et [le système de rendu de Solid](https://www.solidjs.com/docs/latest#rendering).
+
+## Penser Solid
+
+Le design de Solid a été créé avec une certaines opinions sur les principes et les valeurs qui nous aident à créer de meilleurs sites internet et applications. Il est plus simple d'apprendre et d'utiliser Solid quand vous connaissez la philosophie derrière son design.
```
Comment (lukechu10) [inline fr/guides/getting-started.md]:
```suggestion
Le design de Solid a été créé avec certaines opinions sur les principes et les valeurs qui nous aident à construire de meilleurs sites internet et applications. Il est plus simple d'apprendre et d'utiliser Solid quand vous connaissez la philosophie derrière son design.
```

File: fr/guides/getting-started.md [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+
+### 1. Données déclaratives
+
+Les "données déclaratives" est le concept de lié la description des données au comportement de leurs déclarations. Cela permet de facilement grouper le comportement des données dans un seul endroit.
```
Comment (lukechu10) [inline fr/guides/getting-started.md]:
```suggestion
Les "données déclaratives" est le concept de lier la description des données au comportement de leurs déclarations. Cela permet de facilement grouper le comportement des données dans un seul endroit.
```

File: fr/guides/getting-started.md [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+
+-----
+
+## Web Components
+
+Solid est né d'un désire de mettre les Web Components en tant qu'acteur principal. Au fur et à mesure du temps, le design de Solid a évolué et ses objectifs ont changé. Cependant, Solid reste une bonne alternative pour les auteurs de Web Components. [Solid Element](https://github.com/solidjs/solid/tree/main/packages/solid-element) permet d'écrire et d'enrobé des fonctions Composants en Solid pour produire des Web Components petits et performant. Au sain d'une application en Solid, Solid Element est capable d'utiliser l'API de Contexte et les Portails sont supportés dans le Shadow DOM avec l'isolation de style.
+
+## Rendu côté serveur
+
+Solid propose une solution dynamique de rendu côté serveur qui permet une vraie expérience isomorphe. Avec l'utilisation de la primitive "Resource", les requêtes asynchrone sont facilement faite et, plus important, automatiquement sérialiser et synchroniser entre le client et le navigateur.
```
Comment (lukechu10) [inline fr/guides/getting-started.md]:
```suggestion
Solid propose une solution dynamique de rendu côté serveur qui permet une vraie expérience isomorphe. Avec l'utilisation de la primitive "Resource", les requêtes asynchrone sont facilement faite et, plus important, automatiquement sérialisé et synchronisé entre le client et le navigateur.
```

File: fr/guides/getting-started.md [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+        onCleanup(() => clearInterval(timer));
+        return html`<div>${count}</div>`;
+      };
+      render(App, document.body);
+    </script>
+  </body>
+</html>
+```
+
+N'oubliez pas, vous aurez quand même besoin de la librairie utilisez pour l'expression du DOM pour que cela fonctionne avec TypeScript. Vous pouvez utiliser les Tagged Template Literals avec [Lit DOM Expressions](https://github.com/ryansolid/dom-expressions/tree/main/packages/lit-dom-expressions) ou HyperScript avec [Hyper DOM Expressions](https://github.com/ryansolid/dom-expressions/tree/main/packages/hyper-dom-expressions).
```
Comment (lukechu10) [inline fr/guides/getting-started.md]:
```suggestion
N'oubliez pas, vous aurez quand même besoin de la librairie utilisé pour l'expression du DOM pour que cela fonctionne avec TypeScript. Vous pouvez utiliser les Tagged Template Literals avec [Lit DOM Expressions](https://github.com/ryansolid/dom-expressions/tree/main/packages/lit-dom-expressions) ou HyperScript avec [Hyper DOM Expressions](https://github.com/ryansolid/dom-expressions/tree/main/packages/hyper-dom-expressions).
```

**Review by lukechu10 — COMMENTED** [review]
Juste quelques suggestions

File: fr/guides/getting-started.md (hunk 2 above)
Comment (amoutonbrady) [reply fr/guides/getting-started.md]: Je suis relativement d'accord avec le fait de remplacer "créer" par "construire", ça évite notamment la répétition du verbe "créer". Je me demande même si on ne pourrait pas utiliser le verbe "penser" ou "concevoir" plutôt que "créer" au début.

File: fr/guides/getting-started.md (hunk 1 above)
Comment (amoutonbrady) [reply fr/guides/getting-started.md]: J'ai pas vraiment d'opinion. Les deux formulations me semblent plutôt correctes :)

**Comment by amoutonbrady** [comment]
Je suis loin d'être le meilleur en français, j'ai une tendance à faire pas mal de fautes. Ceci étant dit, je pense qu'une fois que toutes les traductions seront faites, je passerai les fichiers MD via https://www.antidote.info/. J'ai une license qui traîne, que j'utilise principalement pour ne pas me ridiculer dans mails pro 😅

Encore une fois, bravo pour le travail accompli !

**Comment by excessivecoding** [comment]
Finished processing suggestions, thanks @lukechu10 & @amoutonbrady 👍

I've used a type some writing assistant on the all the files to catch some grammar issue, but I'm sure you will find some error if you search deeper (i am definitely better with code than french somehow 😅)

Hope this can at least be a good foundation for the French documentation and help to push Solid.js forward in France 🔥

**Comment by davedbase** [comment]
Thanks @xstevenyung. Merging it in now as @MidouWebDev would like to proceed with the final two MDs. If either of you have time the two other translation projects we have are: translating the website (a small task) and the tutorials (a pretty big task). I hate to ask for more contribution but this sort of work does take a lot of hands to get done =)

**Comment by MidouWebDev** [comment]
@davedbase no problem for asking for contributions, we are here for that. Be sure that i am going to contribute to the translation of the website + the two other MD files.

**Pull request merged**